src/panels/lovelace/views/hui-masonry-view.ts

```diff
@@ -261,9 +261,10 @@ export class MasonryView extends LitElement implements LovelaceViewElement {
       :host {
         display: block;
         background: var(--lovelace-background);
-        padding-top: 4px;
-        height: 100%;
+        padding-top: 100px;
+        height: calc(100% + 160px);
         box-sizing: border-box;
+        margin-top: -96px;
```

Comment:
Why is this needed?

Reply:
This styling is what makes the view's background extend behind the header. Without this if you were to use a transparent background on app-header it would show empty space behind it rather than the view's background.

Reply:
This feels very weird. We should just apply the background on the element that contains the tabs + view instead of mushing the view behind the tab bar. And instead of `margin-top` one should just go then for position absolute and just tag it to top, left, right, bottom all 0. But really, just setting the background on the parent is the best approach.

Reply:
So, we should just be setting the background on `ha-app-layout` and dropping the styling changes I made on masonry-view and panel-view.

Reply:
Sorry, guess I'm still thinking of things a bit in terms of a custom addon as this is exactly something I'd do for CH.

Reply:
Removed background and styling changes from `hui-masonry-view` and `hui-panel-view` and added the background to `ha-app-layout`.